src/lib.rs

```diff
@@ -231,44 +231,54 @@ fn rewrite_let(i: &mut Local) {
 
 fn rewrite_match(i: &mut ExprMatch) {
     i.attrs.retain(|attr| !path_eq(&attr.path, "bitmatch"));
-    let mut cases = Vec::with_capacity(i.arms.len());
+    let mut patterns = Vec::with_capacity(i.arms.len());
     let mut cubelist = CubeList::new();
-    for arm in &i.arms {
-        match pat_str(&arm.pat) {
-            Some(s) => {
+    let mut has_wildcard = false;
+    for arm in i.arms.iter() {
+        match pattern(&arm.pat) {
+            Pattern::Str(s) => {
                 if arm.guard.is_none() {
                     cubelist = cubelist.merge(&CubeList::from_list(&[to_cube(&s)]));
                 }
-                cases.push(s);
+                patterns.push(Some(s));
             }
-            None => {
+            Pattern::Wildcard => {
+                has_wildcard = true;
+                patterns.push(None);
+            }
+            Pattern::Invalid => {
                 let pat = &arm.pat;
                 panic!(
-                    "#[bitmatch] match patterns must be string literals, but found {}",
+                    "#[bitmatch] match patterns must be string literals or a wildcard, but found {}",
                     quote! { #pat }
                 )
             }
         }
     }
-    if !true_cubelist(&cubelist) {
-        panic!("Some cases not covered. Cases: {:?}", cases);
+    if !has_wildcard && !true_cubelist(&cubelist) {
```

Comment:
This reads as clearer to me:
```suggestion
    if !(has_wildcard || true_cubelist(&cubelist)) {
```